chicagoland.pandemicresponsecommons.org/manifest.json

```diff
@@ -10,7 +10,7 @@
     "aws-es-proxy": "quay.io/cdis/aws-es-proxy:0.8",
     "covid19-etl": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-etl:4.11.1",
     "covid19-notebook-etl": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-notebook-etl:4.9.4",
-    "covid19-bayes": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-bayes:4.11.0",
+    "covid19-bayes": "quay.io/cdis/covid19-bayes:4.12.0",
```

Comment:
if you want you can push the image to ECR via qaplanetv1 and keep using the ECR PATH